shell/platform/windows/cursor_handler_unittests.cc

```diff
@@ -76,7 +76,6 @@ class CursorHandlerTest : public WindowsTest {
 
     window_ = window.get();
     EXPECT_CALL(*window_, SetView).Times(1);
-    EXPECT_CALL(*window_, GetRenderTarget).WillOnce(Return(nullptr));
```

Comment:
Should this expect a call to `GetWindowHandle`?

Reply:
This mock used to be necessary but the implementation must have changed to no longer need it. 

I've verified that the tests pass and don't output unexpected warnings without this mock. I also updated additional tests that no longer need this mock.